src/backend/router.ts

```diff
@@ -1,13 +1,26 @@
 import path from 'path';
 
-import { Response, Request, Router } from 'express';
+import { Response, Request, Router, NextFunction } from 'express';
 
 import { Client, ensureAuthenticated, envVar, logRequest } from '@navikt/familie-backend';
 import { LOG_LEVEL } from '@navikt/familie-logging';
 
 import { buildPath } from './config';
 import { prometheusTellere } from './metrikker';
 
+export const redirectHvisInternUrlIPreprod = () => {
```

Comment:
Ønsker å redirecte fra intern til ansatt FØR vi sjekker gyldig token for at folk skal slippe å måtte endre alle bookmarks samt "bakoverkompabilitet".